tests/summ_ai_api/summ_ai_test.py

```diff
@@ -228,3 +452,180 @@ async def test_summ_ai_error_handling(
             "Please try again later or contact an administrator",
             response,
         )
+
+
+def test_validate_response():
+    """
+    Test for validate_response method.
+    Tests if return is True, when 'translated_text' is found in request and exception is thrown when not.
+    """
+    response_data_translation_succeeded = {
+        "dummy": "dummy",
+        "translated_text": "Das ist einfaches Deutsch.",
+    }
+    assert (
+        SummAiApiClient(MockedRequest(), PageTranslationForm).validate_response(
+            response_data_translation_succeeded, 200
+        )
+        is True
+    ), "if translated_text found in reponse-data, validate_response should return True"
+
+    response_data_translation_failed = {
```

Comment:
I think it could be 2 separate cases, because there are basically 2 cases: one positive and one negative.
The problem with combining them is that if the first assertion fails, the second one won't be reached.